packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js

```diff
@@ -35,22 +43,39 @@ export default class QuanticSearchBox extends LightningElement {
   input;
   /** @type {import("lwc").HTMLElementTheGoodPart} */
   combobox;
-
+  /** @type {HTMLButtonElement} */
+  clearButton;
```

Comment:
Do you really need both `clearButton` and `closeButton`? You seem to be applying the same changes to both anyways, so it would remove some duplication.

Reply:
You're right it works using only `clearButton`. For some reason I thought i needed to use two elements. Thanks!